python/cudf/cudf/core/dataframe.py

```diff
@@ -3722,6 +3722,49 @@ def sort_values(
             keep_index=not ignore_index,
         )
 
+    def agg(self, aggs):
```

Comment:
The function signature here should be updated to match the Pandas function signature: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.agg.html

It looks like `axis` isn't supported yet, so we should throw a `NotImplementedError` if someone sets it to an unsupported value.